woocommerce/woocommerce-ios
# [Deposits Summary] UI update

Closes: #11528


## Description
This PR updates the UI for `Deposits Summary` in order to sync it with Android and the web, mostly removing a couple of UI items, as well as repositioning bits. We've also removed the property regarding `nextDeposit` since this is no longer necessary as we’re not estimating future deposits based on how long funds pend for. 

## Changes:
- Remove _"x deposits"_ that is displayed underneath the Pending funds balance. 
- Remove existing calendar icon
- Replace "LAST" wording with new calendar icon
- Move "Funds become available"... string before the separator, so is part of the "upper" block.
- Move "Available funds..."... string before the separator, so is part of the "middle" block.
- DEPOSITS heading renamed to LAST DEPOSIT as we’re no longer showing an upcoming estimate.
- Add a separator to the bottom of the component
<img width="300" alt="Screenshot 2023-12-25 at 09 28 09" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/1730750a-a53c-4530-9372-292571c6f6a8">

## Testing instructions


Note that for stores with only one currency, no currencies will appear at the top. As well as you may have no incoming or unknown deposits in your store and this will be reflected in the UI. Let me know if you want access to a multi-currency store with set deposits for ease of testing :)

1. On a store with WooPayments enabled go to Menu > Payments
2. Observe deposits being loaded at the top of the screen, tap the chevron to expand Deposit Summary details.
3. Observe that the UI resembles the `after` screenshot.

## Screenshots

| Before | After |
|--------|--------|
| ![simulator_screenshot_1602AD82-1033-4DDE-AD4C-3D6799FEE037](https://github.com/woocommerce/woocommerce-ios/assets/3812076/7502a6e0-23db-4245-8747-6dfc80b284cf) | ![simulator_screenshot_827A5A60-52B1-4D43-BFC9-7DEB86F01EB7](https://github.com/woocommerce/woocommerce-ios/assets/3812076/284e3cb5-ef3b-421a-a362-c486d4b16960) | 
---
